src/collective/faq/browser/static/faq.css

```diff
@@ -60,3 +60,6 @@
     border: thin dotted;
     background-color: #EFF7FF;
 }
+h2 .faq-nested-title {
+    font-weight: normal !important;
```

Comment:
@rodfersou is the !important here really necessary? Can't we just use a more specific CSS selector? Using !important is an anti-pattern and should be avoided.